### DISCUSSION TITLE
Is it possible to record video streams without decoding them?

### DISCUSSION THREAD

**Discussion by BellaCoola** [blakeblackshear/frigate#4870] [Ask A Question] [answered]
Hello,

If I wanted to use Frigate only as a software for managing CCTV cameras and recording their video streams - without need for object detection / motion detection etc. is it possible to record the incoming video streams without decoding them? I know this is possible in some other software packages, such as Shinobi (when using its copy mode) and it seems nice, since in such case only very little hardware resources are utilized. When it comes to Frigate, it seems that in the past this wasn't possible (https://github.com/blakeblackshear/frigate/issues/3753), but with the release of 0.12 beta, could such setup be possible? (I haven't found a definitive answer in the documentation.)

Thank you very much and have a nice day.

**Comment by NickM-27** [comment, marked as answer]
No this still isn't possible. It's on the roadmap to be able to disable the detect stream when not in use.

**Reply by BellaCoola** [reply]
Thank you very much. If I understand this correctly, it is now (since 0.12 beta) possible to record the video streams without need for the decoding, but the issue is that it is currently not possible to disable the detect stream, which still requires the decoding to happen (and which would be redundant in my case since I don't want to perform any detection)?

**Reply by NickM-27** [reply]
Yes recording does not do any decoding or anything. However, decoding is required for the detect stream which is always running even if object and motion detection is disabled.

**Reply by BellaCoola** [reply]
Please is there some issue already present for this feature (ability to disable the detect stream), so I can check its status? I tried to look for such issue here on GitHub, but unfortunately I haven't found it.

**Reply by NickM-27** [reply]
It would be covered under https://github.com/blakeblackshear/frigate/issues/1911

**Reply by BellaCoola** [reply]
Great, thank you very much :slightly_smiling_face:

**Reply by BellaCoola** [reply]
@NickM-27 since the https://github.com/blakeblackshear/frigate/issues/1911 is resolved now, please is it now possible to record video streams without decoding them? I haven't found such information anywhere so far.

**Reply by NickM-27** [reply]
No, after 1911 was implemented and more discussion has taken place, there are too many places in the WebUI and various features that depend on the decoded stream. The vast majority of hardware that Frigate can run on supports basic decoding.

In most cases this is detected automatically meaning that hardware acceleration will be used for decoding, and object detection is disabled by default, so this should be workable for the use case you have described.
